broker/structs/structs.go

```diff
@@ -41,9 +41,9 @@ type ClusterState struct {
 	SuperuserCredentials PostgresCredentials `json:"superuser_credentials"`
 	AppCredentials       PostgresCredentials `json:"app_credentials"`
 	AllocatedPort        int                 `json:"allocated_port"`
-	Nodes                []*Node             `json:"nodes"`
 	SchedulingInfo       SchedulingInfo      `json:"info"`
 	ServiceInstanceName  string              `json:"service_instance_name"`
+	Nodes                []*Node             `json:"nodes"`
```

Comment:
goimports will reorder this alphabetically each time (I press save on this file)